code/game/machinery/suit_storage_unit.dm

```diff
@@ -663,14 +663,14 @@ GLOBAL_LIST_EMPTY(suit_cycler_typecache)
 		/datum/suit_cycler_choice/department/emag
 	)
 	can_repair = 1
-
+/* //VOREStation Removal
 /obj/machinery/suit_cycler/exploration

```

Comment:
Exploration voidsuits are not going away, these should be kept, just moved into the gateway or something.